admin/views/tool-import-export.php

```diff
@@ -93,20 +93,16 @@
 ];
 
 ?>
-	<br/><br/>
 
-	<h2 class="nav-tab-wrapper" id="wpseo-tabs">
+	<nav class="yoast-tabs" id="wpseo-tabs">
+		<ul class="yoast-tabs__list">
 		<?php foreach ( $tabs as $identifier => $tab ) : ?>
-			<a class="nav-tab" id="<?php echo esc_attr( $identifier . '-tab' ); ?>" href="<?php echo esc_url( '#top#' . $identifier ); ?>"><?php echo esc_html( $tab['label'] ); ?></a>
+			<li class="yoast-tabs__list-item">
+			<a class="yoast-tabs__list-item-link" id="<?php echo esc_attr( $identifier . '-tab' ); ?>" href="<?php echo esc_url( '#top#' . $identifier ); ?>"><?php echo esc_html( $tab['label'] ); ?></a>
+			</li>
 		<?php endforeach; ?>
-
-		<?php
-		/**
-		 * Allow adding a custom import tab header.
-		 */
-		do_action( 'wpseo_import_tab_header' );
```

Comment:
This action hook can't be removed. It is used to add more import "tabs". Premium for example adds a few more tabs, and SEP Local does the same.

Import tabs in Premium:
<img width="1043" alt="Screenshot 2020-05-14 at 12 06 55" src="https://user-images.githubusercontent.com/1682452/81922779-e543f000-95dc-11ea-996f-9752e86af2b6.png">

Also: note that the action callbacks on Premium and Local add tabs markup that contains only a link `<a>` element. They don't add a `<li>` so the markup will break.